privateSBOMExchange/Evaluation/README.md

```diff
@@ -0,0 +1,102 @@
+# Petra Evaluation Guide
+
+This guide provides step-by-step instructions for evaluating **Petra** using SBOM dataset.  
+You will download an SBOM corpus, preprocess it, run the evaluation benchmarks, and generate performance plots.
+
+## 1. Download SBOM Dataset
+
+Choose **one** of the following options to obtain the bom-shelter dataset.
+
+### Option 1 – Clone bom-shelter directly
+```bash
+cd SBOMCTL/sbom-data
+git clone https://github.com/chainguard-dev/bom-shelter
+```
+
+### Option 2 – Use git submodules
+If your repository is already configured with a submodule reference:
+```bash
+cd SBOMCTL/sbom-data
+git submodule init
+git submodule update
+```
+    
+### Option 3 – Download from Zenodo
+Download the bom-shelter from zenodo into `SBOMCTL/sbom_data` directory:
+https://doi.org/10.5281/zenodo.17859760
+   
+
+
+## 2. Setup Petra 
+
+You can set up a virtual environment in two ways:
+
+```bash
+make init
+```
+
+or
+
+```bash
+virtualenv <venv name>
+source <venv name>/bin/activate
+pip install -r requirements.txt
```

Comment:
pandas is still hardcoded as `pandas==2.2.2` this may not work in all environments.